controllers/operatorscaler_controller.go

```diff
@@ -218,7 +217,8 @@ func (r *OperatorScalerReconciler) reconcileMetrics(ctx context.Context, logger
 					crItem.Name+"-storagesystem",
```

Comment:
```suggestion
					crItem.Name,
```

We also want to change this and remove the `-storagesystem` as it wont be present anymore. @bipuladh pls take a look.

Reply:
Yes please remove it